src/internal/clientsdk/pfs.go

```diff
@@ -147,6 +147,48 @@ func ListCommit(client pfs.API_ListCommitClient) ([]*pfs.CommitInfo, error) {
 	return results, nil
 }
 
+func ForEachFindCommitsResponse(client pfs.API_FindCommitsClient, cb func(response *pfs.FindCommitsResponse) error) error {
+	for {
+		x, err := client.Recv()
+		if err != nil {
+			if err == io.EOF {
+				break
+			}
+			return errors.EnsureStack(err)
+		}
+		if err := cb(x); err != nil {
+			if errors.Is(err, pacherr.ErrBreak) {
+				err = nil
+			}
+			return err
+		}
+	}
+	return nil
+}
+
+type FindCommitsResponseUser struct {
```

Comment:
This seems useful to me, probably worth reviving `clientsdk` for.

Reply:
This can probably live in the client package.